static/app/components/core/alert.chonk.tsx

```diff
@@ -0,0 +1,130 @@
+import type {AlertProps} from 'sentry/components/alert';
+import {chonkStyled, useChonkTheme} from 'sentry/utils/theme/theme.chonk';
+import type {ChonkPropMapping} from 'sentry/utils/theme/withChonk';
+
+const chonkAlertPropMapping: ChonkPropMapping<AlertProps, ChonkAlertProps> = props => {
+  return {
+    ...props,
+    type:
+      props.type === 'muted' ? 'subtle' : props.type === 'error' ? 'danger' : props.type,
+  };
+};
+
+interface ChonkAlertProps extends Omit<AlertProps, 'type'> {
+  type: 'subtle' | 'info' | 'warning' | 'success' | 'danger';
+  size?: 'sm';
+}
+
+function AlertPanel({type, size, ...props}: ChonkAlertProps) {
+  const theme = useChonkTheme();
+
+  return (
+    <AlertPanelDiv
+      {...props}
+      colors={chonkAlertTheme(type, theme)}
+      padding={chonkAlertPadding(size, theme)}
+    >
+      {props.children}
```

Comment:
nit: we should probably destruct `children` to get them out of the `...props` spreading, or remove the explicit `props.children` passing here, as they are already part of the `...props` spreading.